test/enhancer-test.js

```diff
@@ -1,37 +1,41 @@
-"use strict";
+'use strict';
 
 const React = require('react');
 const renderToString = require('react-dom/server').renderToString;
 
 const Enhancer = require('../lib/enhancer');
-const StyleRoot = require('../lib/components/style-root');
 const createEsClass = require('../lib/test-helpers').createEsClass;
 
 const render = (Component, props) =>
-  renderToString(React.createElement(Component, Object.assign({
-    radiumConfig: { userAgent: 'testy-mctestface' }
-  }, props)));
+  renderToString(
+    React.createElement(
+      Component,
+      Object.assign(
+        {
+          radiumConfig: {userAgent: 'testy-mctestface'}
+        },
+        props
+      )
+    )
+  );
 
 // Full assertion wrapper
-const assertValidEnhancedComponent = (Composed) => {
+const assertValidEnhancedComponent = Composed => {
   const Enhanced = Enhancer(Composed);
   const rendered = render(Enhanced);
 
   expect(render(Composed)).to.not.equal(rendered);
-  expect(rendered)
-    .to.contain('data-radium="true"').and
-    .to.contain('style="background:red;color:white"');
+  expect(rendered).to
```

Comment:
Is this an auto-formatted result?  Chai assertions are typically always:

```js
expect(foo)
  .to.SOMETHING.and
  .to.SOMETHING_ELSE.and
  .to.LAST_THING;
```

Reply:
Or is that prettier turned loose?

Reply:
looks like that is what `prettier` wants. `npm run lint` will fail otherwise.